magento/magento2
# fixed #24234

### Description (*)

If set quantity as decimal and set local in which comma use as seperator. Then decimal display in minicart. I used Zend_Locale_Format to convert number according to locale.

### Fixed Issues (if relevant)

1. magento/magento2#24234 : Incorrect decimal separator in minicart and checkout.

### Manual testing scenarios (*)

1. Set local in which comma use as number seperator for example de_DE.
2. Create product with decimal quantity allow.
3. Add to cart same product with decimal quantity.
4. Now check minicart.

### Questions or comments


### Contribution checklist (*)
